<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mActionZoomIn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionZoomIn.svg</normaloff>:/images/images/themes/default/mActionZoomIn.svg</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="mActionZoomOut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionZoomOut.svg</normaloff>:/images/images/themes/default/mActionZoomOut.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mActionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionPan.svg</normaloff>:/images/images/themes/default/mActionPan.svg</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="mActionZoomFullExtent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionZoomFullExtent.svg</normaloff>:/images/images/themes/default/mActionZoomFullExtent.svg</iconset>
   </property>
   <property name="text">
    <string>Full Extent</string>
   </property>
  </action>
  <action name="mActionZoomLast">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionZoomLast.svg</normaloff>:/images/images/themes/default/mActionZoomLast.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Last</string>
   </property>
  </action>
  <action name="mActionZoomNext">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionZoomNext.svg</normaloff>:/images/images/themes/default/mActionZoomNext.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Next</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/exit.svg</normaloff>:/images/images/themes/default/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="mActionMeasure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionMeasure.svg</normaloff>:/images/images/themes/default/mActionMeasure.svg</iconset>
   </property>
   <property name="text">
    <string>Measure Dist</string>
   </property>
  </action>
  <action name="mActionMeasureArea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionMeasureArea.svg</normaloff>:/images/images/themes/default/mActionMeasureArea.svg</iconset>
   </property>
   <property name="text">
    <string>Measure Area</string>
   </property>
  </action>
  <action name="mActionMeasureAngle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/mActionMeasureAngle.svg</normaloff>:/images/images/themes/default/mActionMeasureAngle.svg</iconset>
   </property>
   <property name="text">
    <string>measure Angle</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/help.svg</normaloff>:/images/images/themes/default/help.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionNavigateTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/navigate.svg</normaloff>:/images/images/themes/default/navigate.svg</iconset>
   </property>
   <property name="text">
    <string>NavigateTool</string>
   </property>
  </action>
  <action name="actionMeasureTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/measure.svg</normaloff>:/images/images/themes/default/measure.svg</iconset>
   </property>
   <property name="text">
    <string>MeasureTool</string>
   </property>
   <property name="toolTip">
    <string>MeasureTool</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/setting.svg</normaloff>:/images/images/themes/default/setting.svg</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="toolTip">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionHistoryTool">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/images/images/themes/default/history.svg</normaloff>:/images/images/themes/default/history.svg</iconset>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="toolTip">
    <string>History</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
